packages/react/src/components/Nav/Nav.styles.ts

```diff
@@ -9,6 +9,12 @@ import {
 import type { INavStyleProps, INavStyles } from './Nav.types';
 import type { IButtonStyles } from '../../Button';
 
+// The number pixels per indentation level for Nav links.
+const _indentationSize = 14;
+
+// The number of pixels of left margin
+const _baseIndent = 3;
```

Comment:
nit (so it's a little easier to understand within this file)
```suggestion
/** The number pixels per indentation level for Nav links. */
const _indentationSize = 14;

/** The number of pixels of left margin */
const _baseIndent = 3;
```